app/controllers/uploaded_images_controller.rb

```diff
@@ -0,0 +1,33 @@
+class UploadedImagesController < ApplicationController
+  layout "site", :except => :rss
+  before_action :authorize_web
+  before_action :set_locale
+
+  authorize_resource 
+
+  before_action :lookup_user, :only => [:show, :comments]
+  before_action :allow_thirdparty_images, :only => [:new, :create, :edit, :update, :index, :show, :comments]
+
+  def index
+    if params[:display_name]
+      @user = User.active.find_by(:display_name => params[:display_name])
+      if @user
+        @title = @user.display_name + "'s uploaded images"
+      end
+    end
+    @images = UploadedImage.get_uploaded_images(current_user)
+  end
+
+  def destroy_multiple
+    if params[:deleted_img_ids]
+      for id in params[:deleted_img_ids] do
+        uri = URI('http://noter-backend:3001/api/v0.1/images/' + id.to_s + '/')
```

Comment:
Is this guaranteed to always work?
Wouldn't it be better to have a config parameter with API URI?